notice_and_comment/static/regulations/css/less/fonts.less

```diff
@@ -0,0 +1,29 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.less
+ */
+
+@font-face {
```

Comment:
Because Light and Semibold are separate font files, we need to uniquely declare them even if you set a specific weight attribute to it.

If we were to include Google's stylesheet, it still does the approximately the same thing: https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,600

However, you're right in setting font-family fallbacks. I didn't put a generic serif after either of the Source Sans declarations. Let me know what you suggest there.